tests/Client/OfficeAddIn/AnnotationTable/SuccessfulTests.fs

```diff
@@ -0,0 +1,128 @@
+
+open Fable.Core
+open Fable.Core.JsInterop
+open Fable.Mocha
+
+open ExcelJS.Fable.Excel
+open Shared.ARCtrlHelper
+
+open ARCtrl
+// ⚠️ These tests do only work on node.js as the we sents user data to microsoft !!! ⚠️
+// Check out https://github.com/OfficeDev/Office-Addin-Scripts/issues/905
+
+// 💡 Read More
+// - https://learn.microsoft.com/en-us/office/dev/add-ins/testing/unit-testing
+// - RequestContext object: https://learn.microsoft.com/en-us/javascript/api/excel/excel.requestcontext?view=excel-js-preview
+
+let compositeColumn =
+    let header = CompositeHeader.Parameter (OntologyAnnotation.create("Test"))
+    CompositeColumn.create(header)
+
+let arcTable =
+    let table = ArcTable.init("annotationTable 2")
+    let header = CompositeHeader.Component (OntologyAnnotation.create("TestI"))
+    let body = CompositeCell.emptyData
+    ArcTable.addColumnFill(header, body) table
+
+let arcInvestigation =
+    let investigation = ArcInvestigation.init("New Investigation")
+    ArcFiles.Investigation investigation
+
+let arcAssay =
+    let assay = ArcAssay.init("New Assay")
+    assay.AddTable(arcTable)
+    ArcFiles.Assay assay
+
+let arcStudy =
+    let study = ArcStudy.init("New Study")
+    study.AddTable(arcTable)
+    ArcFiles.Study (study, [])
+
+let arcTemplate =
+    let template = Template.init("New Template")
+    template.Table <- arcTable
+    ArcFiles.Template template
+
+let private TestsBasic = testList "Basic tests" [
+    testCaseAsync "develop mock" <| async {
+        let testContext: RequestContext = importDefault "../../Fixtures/OfficeMockObjects/BasicFixture.js"
+        let! result = OfficeInterop.ExcelUtil.getSelectedRangeAdress testContext |> Async.AwaitPromise
+        Expect.equal result "C2:G3" "Verify correct setup"
+    }
+]
+
+let private TestsSuccessful = testList "Successful tests" [
+    testCaseAsync "develop getTablesTest successful" <| async {
+        let testContext: RequestContext = importDefault "../../Fixtures/OfficeMockObjects/AnnotationTableFixtureSuccessful.js"
+        let! resultRes = OfficeInterop.Core.getExcelAnnotationTables testContext |> Async.AwaitPromise
+
+        let result = Expect.wantOk (Result.Ok resultRes) "This is a message"
+        Expect.equal (Result.toOption result).Value.Count 1 "This is a message"
+    }
+
+    testCaseAsync "develop rectifyTermColumnsTest successful" <| async {
+        let testContext: RequestContext = importDefault "../../Fixtures/OfficeMockObjects/AnnotationTableFixtureSuccessful.js"
+        let getTerms (termNames:string list) =
+            promise {
+                return [|None|]
+            }
+        let! resultRes = OfficeInterop.Core.Main.rectifyTermColumns (testContext, getTerms) |> Async.AwaitPromise
+
+        let result = Expect.wantOk (Result.Ok resultRes) "This is a message"
+        Expect.equal result.Head.MessageTxt "The annotation table annotationTable 1 is valid" "This is a message"
```

Comment:
Do not pass `"This is a message"` as third argument for all Expect functions. This message will be shown should the test fail and should be written to be useful in this case.